build-system/tasks/check-links.js

```diff
@@ -84,6 +84,25 @@ function checkLinks() {
 }
 
 /**
+ * Filters out markdown elements that contain localhost links.
+ *
+ * @param {string} markdown Original markdown.
+ * @return {string} Markdown after filtering out localhost links.
+ */
+function filterLocalhostLinks(markdown) {
+  var localhostPattern = 'http:\/\/localhost:8000';
```

Comment:
can be merged into a single regex, feel free to add TODO + issue to tackle soon.